cvat/apps/iam/schema.py

```diff
@@ -39,22 +39,13 @@ class TokenAuthenticationScheme(TokenScheme):
     priority = 0
     match_subclasses = True
 
-    def get_security_requirement(self, auto_schema):
-        # These schemes must be used together
-        return {"sessionAuth": [], "csrfAuth": [], self.name: []}
-
     def get_security_definition(self, auto_schema):
         schema = super().get_security_definition(auto_schema)
         schema["x-token-prefix"] = self.target.keyword
         schema["description"] = textwrap.dedent(
-            f"""
-            To authenticate using a token (or API key), you need to have 3 components in a request:
-            - the 'sessionid' cookie
-            - the 'csrftoken' cookie or 'X-CSRFTOKEN' header
-            - the 'Authentication' header with the '{self.target.keyword} ' prefix
-
+            f"""\
             You can obtain an API key (the token) from the server response on
-            the basic auth request.
+            the basic auth request in the /api/auth/login/ endpoint.
```

Comment:
```suggestion
            the /api/auth/login/ endpoint.
```

This endpoint does not accept basic authentication.

Reply:
Do you mean it doesn't use the `basicAuth` schema defined nearby? You still have to use basic auth credentials to make authenticate via this endpoint.

Reply:
> Do you mean it doesn't use the `basicAuth` schema defined nearby?

Yes. It takes credentials in the request body instead.

> You still have to use basic auth credentials to get a token via this endpoint.

Sure, the credentials are the same, but the protocol is different.